packages/generator/src/edmx-to-vdm/common/enum-type.ts

```diff
@@ -1,7 +1,14 @@
+import { createLogger, unique } from '@sap-cloud-sdk/util';
+import BigNumber from 'bignumber.js';
 import { ServiceNameFormatter } from '../../service-name-formatter';
 import { VdmEnumType } from '../../vdm-types';
 import { EdmxEnumType } from '../../edmx-parser/v4';
 
+const logger = createLogger({
+  package: 'generator',
+  messageContext: 'enum-type'
+});
+
 export function transformEnumTypesBase(
   edmTypes: EdmxEnumType[],
```

Comment:
`edmTypes` is probably a copy paste error, I think it should be `edmxEnumTypes`.